### PR TITLE
LPS-157718 add checking for driverClassName in SetupWizardUtil._testConnection

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

This is for ticket https://issues.liferay.com/browse/LPS-157718.
Thanks for checking.

### CODE REVIEW

File: portal-impl/src/com/liferay/portal/setup/SetupWizardUtil.java [hunk 1]
```diff
@@ -511,4 +518,15 @@ private static boolean _writePropertiesFile(
 	private static final Log _log = LogFactoryUtil.getLog(
 		SetupWizardUtil.class);
 
+	private static final Set<String> _driverClassNameWhiteList =
+		SetUtil.fromArray(
+			new String[] {
+				"com.ibm.db2.jcc.DB2Driver", "org.hsqldb.jdbc.JDBCDriver",
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/setup/SetupWizardUtil.java:524]: @jeffwu0724 Please sort the driver class name.

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
@jeffwu0724 is needed to run relevant again?

**Comment by jeffwu0724** [comment]
Hi @catarineaguiar , I only sorted the order of the strings, and didn't change any code logic. Therefore, we can still use the previous relevant test

**Comment by catarineaguiar** [comment]
@jeffwu0724 in this case, no regressions on relevant.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by shuyangzhou** [comment]
This list's values should be from portal.properties. In case of driver upgrades, they might change the class names, it would be painful to have to update the code for it.

**Comment by tinatian** [comment]
@jeffwu0724 Please create a new property in portal.properties and resend.

Thanks.
Tina

**Pull request closed without merging**